python/pyarrow/tests/test_fs.py

```diff
@@ -1052,6 +1052,111 @@ def test_s3_options(monkeypatch):
         )
 
 
+@pytest.mark.s3
+def test_s3_proxy_options(monkeypatch):
+    from pyarrow.fs import S3FileSystem
+
+    # The following two are equivalent:
+    proxy_opts_1_dict = {'scheme': 'http', 'host': 'localhost', 'port': 8999}
+    proxy_opts_1_str = 'http://localhost:8999'
+    # The following two are equivalent:
+    proxy_opts_2_dict = {'scheme': 'https', 'host': 'localhost', 'port': 8080}
+    proxy_opts_2_str = 'http://localhost:8080'
```

Comment:
Should be "https" here. If the test succeeds, then something is wrong :-)

Reply:
Thanks for catching that. We didn't have a test comparing `opts_2_str` and `opts_2_dict` so it didn't fail, but I'll add one.

Reply:
Done.